### ISSUE TITLE
Support for computed fields

### ISSUE THREAD

**Issue by robderijk** [dcasia/nova-inline-morph-to#8]
Hello, it seems like computed fields are not supported by this package.

For example I have this in my embedded resource:

```
Text::make('Name', function () {
    return $this->first_name.' '.$this->last_name;
});
```

First name and last name properties are always null. I searched in the code and there is a new model created (instead of using an existing model) here https://github.com/dcasia/nova-inline-morph-to/blob/master/src/InlineMorphTo.php#L68 which might explain that behaviour but I'm not sure if that's the issue. Do you have an idea?

**Comment by milewski** [comment]
@robderijk in which view are you seeing that `Text` field as null? detail? index? or edit?

**Comment by milewski** [comment]
@robderijk Actually I realized what was the issue, fixed on version [v0.1.8](https://github.com/dcasia/nova-inline-morph-to/releases/tag/v0.1.8)

The issue was that when you create the computed field the initial context is an empty model, but later on when the actual model is fetched the closure context wasnt updated, this was an easy fix: https://github.com/dcasia/nova-inline-morph-to/commit/f5e4ebd26ba448dce2e8010a92972724ea8f41a9
